client/src/main/java/tw/waterball/survive/io/client/view/Globle.java

```diff
@@ -0,0 +1,7 @@
+package tw.waterball.survive.io.client.view;
+
+public class Globle {
```

Comment:
1. It's `Global`
2. Please use relative path instead of absolute path
3. I don't think we need a global constants class, discuss it with @a8901053  :)